pkg/migrations/sql/preinstall/006-set_up_ha.sql

```diff
@@ -0,0 +1,146 @@
+CREATE TABLE SCHEMA_CATALOG.ha_locks
+(
+    cluster_name TEXT PRIMARY KEY,
+    leader       TEXT,
```

Comment:
For better understanding, we should rename `leader` => `leader_label` or `leader_label_name`, so it becomes cluster_name, leader_name.